pdns/dnsdist.hh

```diff
@@ -646,6 +646,16 @@ struct ServerPool
   NumberedVector<shared_ptr<DownstreamState>> servers;
   std::shared_ptr<DNSDistPacketCache> packetCache{nullptr};
   std::shared_ptr<ServerPolicy> policy{nullptr};
+
+  const size_t countServersUp() {
```

Comment:
This should be `size_t countServersUp() const {` instead, the current version is marking the return value `const`, which does not matter since it's neither a pointer nor a reference.

Reply:
Ah, I misunderstood.